mindjunkies/dashboard/views.py

```
@@ -54,17 +54,24 @@ def get(self, request: HttpRequest, status:str) -> HttpResponse:
             "courses": courses,
             "status": "Published",
         }
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)